doc/logging.md

```diff
@@ -0,0 +1,24 @@
+# How does Happy test log work
+
+## Types of Happy logs
+  * Happy host logs — Happy will run as host, Happy host logs will have logs from Happy tool only.
+
+  * Happy device logs — Happy will host virtual network which has multiple virtual devices, Happy device logs will have logs for software running on the virtual device, openweave is one of the software example, so device logs will have openweave logs.
+
+## Where Happy logs are stored
+
+Both Happy host and device logs location are processed in `happy/Driver.py` as below:
+  * If `HAPPY_LOG_DIR` is found in os environment variable, Happy uses `HAPPY_LOG_DIR` value as log folder location to store Happy test logs. For example, openweave-core tests store Happy test logs along with feature folder under build folder.
+  * If `HAPPY_LOG_DIR` is not found in os environment variable, Happy uses `default_happy_log_dir` from [happy/conf/main_config.json](https://github.com/openweave/happy/blob/master/happy/conf/main_config.json) as the default log folder location to store Happy test logs.
```

Comment:
Is this something the user can or should control or modify? What is an example of the default value associated with the key 'default_happy_log_dir'? Is it /tmp/<something>? Is it ${HOME}/<something>?

Reply:
Right, user can modify 'HAPPY_LOG_DIR' if they want.
'default_happy_log_dir' default value is /tmp in main_config.json